catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -552,15 +556,20 @@ public SourceResponse call() throws Exception {
             }
 
             final SourceResponse sourceResponse = source.query(queryRequest);
+=======
+            final SourceResponse sourceResponse =
```

Comment:
Might want to get rid of sourceResponse to avoid confusion in the future..  Ie, 
`final SourceResponse clonedSourceResponse = cloneResponse( source.query(new QueryRequestImpl(request.getQuery(), request.getProperties())));`